src/ogr/geometry.jl

```diff
@@ -20,7 +20,7 @@ function fromWKB(data)
     return IGeometry(geom[])
```

Comment:
My main question is how best to set `OSRAxisMappingStrategy` for these spatial refs

Reply:
You'd replace the `C_NULL` in `C_NULL::GDALSpatialRef,` with the corresponding [spatialref that you want to assign to the created geometry object](https://github.com/JuliaGeo/GDAL.jl/blob/e55555527ceddbf8f2a856ba3a5bf5c046549c51/src/ogr_api.jl#L15).

Reply:
If I recall correctly, the geometries and spatialrefs in GDAL are not super well integrated and can lead to weird gotchas with projections, so I'd be hesitant about such an approach until you're certain about it.

Reply:
Ok I've simplified this to exclude all `convert` methods. It only works on `reproject` and `importCRS`. If someone requires it for geometries it shouldn't be too much work - but I don't need it and don't have time to work around GDAL gotchas.